Data Client/server.js

```diff
@@ -106,8 +119,49 @@ app.get("/sites", jwt.authenticateSessionToken, async (req, res) => {
   }
 });
 
-// Start the server
-const PORT = process.env.PORT || 3000;
-app.listen(PORT, () => {
-  console.log(`Server is running on http://localhost:${PORT}`);
-});
+// Start server with NGROK
+const startServer = async () => {
```

Comment:
Can we make this a standalone script mapped to its own command in package.json (i.e. `npm run ngrok`) ? It'll be easier for local dev in general to just run the server start and ngrok start on two different instances

Reply:
Would we still be able to serve the ngrok link in the terminal so they can easily copy/paste? I wouldn't want to lose this functionality

Reply:
Canva is able to do it - [I'm gonna dig around here and see what we can emulate](https://github.com/canva-sdks/canva-apps-sdk-starter-kit/blob/b49bf55679121374c2a146e8e63a613754f372d1/scripts/start/app_runner.ts)